packages/maskbook/src/extension/background-script/StorageService.ts

```diff
@@ -27,7 +27,9 @@ class MutexStorage<T extends browser.storage.StorageValue> {
                     const stored = await browser.storage.local.get(key)
                     callback(null, (stored ?? {})[key] as T)
                 } catch (e) {
-                    callback(e)
+                    if (e instanceof Error) {
+                        callback(e)
```

Comment:
This change actually changes the semantics and might break the code. Please always re-throw the error